composer.json

```diff
@@ -15,14 +15,15 @@
     ],
     "require": {
         "php": "^8.1",
-        "illuminate/console": "^9.0|^10.0",
-        "illuminate/support": "^9.0|^10.0"
+        "illuminate/console": "^9.0|^10.0|^11.0",
+        "illuminate/support": "^9.0|^10.0|^11.0"
     },
     "require-dev": {
-        "pestphp/pest": "^1.22.3"
+        "pestphp/pest": "^1.22.3|^2.34"
```

Comment:
Feel free to drop support for Pest 1.x, and Laravel 9.x 👍🏻

Reply:
Nice, removed!